examples/linux_asyncio_consumer.py

```diff
@@ -0,0 +1,164 @@
+import argparse
+import asyncio
+import os
+import struct
+import sys
+import weakref
+
+import logging
+import json
+import pprint
+
+from confluent_kafka import Consumer
+from confluent_kafka import KafkaException
+
+
+assert sys.platform == 'linux', "This example is linux only, cause of eventfd"
+
+
```

Comment:
It'd be good with a bit of text here (or after the license) which explains what this example is for, and why.